recipes/mutamr/meta.yaml

```
@@ -0,0 +1,51 @@
+    - python >=3.10
+    - pip
+  run:
+    - python >=3.10
+    - bwa =0.7.18
+    - delly =1.2.8
+    - freebayes =1.3.8
+    - bcftool =1.20
+    - samtools =1.20
+    - samclip =0.4.0
+    - snpeff =5.2
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)